DataDefinitions/Body.cs

```diff
@@ -51,7 +51,7 @@ public class Body
         public string stellarclass;
 
         /// <summary>The Luminosity Class of the Star (since 2.4)</summary>
-        public string luminosityClass { get; set; }
+        public string luminosityclass { get; set; }
```

Comment:
Need to check that changing the case here doesn't break anything

Reply:
all others are in lowercase, so i think that only one with uppercase inside was not very good

Reply:
You can't just go renaming things without checking what refers to that name.